diff --git a/vbetables-gen.c b/vbetables-gen.c
index 550935a..fe2a253 100644
--- a/vbetables-gen.c
+++ b/vbetables-gen.c
@@ -76,6 +76,9 @@ ModeInfo modes[] = {
 { 2560, 1600, 16                     , 0x18a},
 { 2560, 1600, 24                     , 0x18b},
 { 2560, 1600, 32                     , 0x18c},
+{ 1024, 1024, 16                     , 0x18d},
+{ 1024, 1024, 24                     , 0x18e},
+{ 1024, 1024, 32                     , 0x18f},
 { 0, },
 };
 
